images/cli-artifacts/Dockerfile.rhel

```diff
@@ -7,6 +7,6 @@ WORKDIR /go/src/github.com/openshift/origin
 COPY . .
 RUN OS_RELEASE_WITHOUT_LINKS=y OS_BUILD_RELEASE_ARCHIVES=n OS_ONLY_BUILD_PLATFORMS="^(darwin|windows)/amd64$" bash -x hack/build-cross.sh
 
-FROM registry.svc.ci.openshift.org/ocp/4.0:cli
+FROM registry.svc.ci.openshift.org/ocp/4.1:cli
```

Comment:
these should be 4.2 right? you are merging to master.

Reply:
Good catch, updated `cli` usages and replaced `4.1:base` with `4.2:base`